training/ItemTableViewController.swift

```diff
@@ -14,67 +14,29 @@ class ItemTableViewController: UITableViewController {
     let celIdentifier = "ItemTableViewCell"
 
     var work: DispatchWorkItem? = nil
-    var items: [String] = Array(repeating: "", count: 10)
+    var items: [String] = Array(repeating: "", count: 0)
 
     override func viewDidLoad() {
         super.viewDidLoad()
         tableView.rowHeight = UITableViewAutomaticDimension
         tableView.estimatedRowHeight = 44.0
-        generateData()
-        initWork()
-
-        NotificationCenter.default.addObserver(
-                self,
-                selector: #selector(appBecomeActive(_:)),
-                name: NSNotification.Name.UIApplicationDidBecomeActive,
-                object: nil)
-
-        NotificationCenter.default.addObserver(
-                self,
-                selector: #selector(appGoBackground(_:)),
-                name: NSNotification.Name.UIApplicationDidEnterBackground,
-                object: nil)
-
-        triggerTask()
-    }
 
-    override func viewDidDisappear(_ animated: Bool) {
-        NotificationCenter.default.removeObserver(self)
-        work?.cancel()
-        work = nil
-        super.viewDidDisappear(animated)
+        getItems()
     }
 
-    private func initWork() {
-        work = DispatchWorkItem(block: {
-            let randomIndex = Int(randomBelow: 10)
-            self.items[randomIndex] = String(randomSubStringCount: 10, randomStringLength: 10)
-            self.triggerTask()
-
-            NSLog("Trigger task again")
+    private func getItems() {
+        let alert = UIAlertController(title: "Connecting", message: nil, preferredStyle: UIAlertControllerStyle.alert)
+        self.present(alert, animated: true)
```

Comment:
getItems  is called from viewDidLoad so you are presenting alert on view controller which doesn't show up yet?
as a result there is warning on xcode console about detached view controller